src/custom/EchoAdaptSession.php

```diff
@@ -89,17 +90,46 @@ public function getTestMap($results = [])
      * (non-PHPdoc)
      * @see \oat\libCat\CatSession::getResults()
      */
-    public function getResults()
+    public function getResults($scope = self::TEST_RESULTS_SCOPE)
```

Comment:
misleading function name, if only partial results are required (are they?), consider splitting into getItemResults and getTestResults